manage.sh

```diff
@@ -123,9 +123,9 @@ docker_build() {
     SEARX_GIT_VERSION=$(git describe --match "v[0-9]*\.[0-9]*\.[0-9]*" HEAD 2>/dev/null | awk -F'-' '{OFS="-"; $1=substr($1, 2); if ($3) { $3=substr($3, 2); }  print}')
 
     # add the suffix "-dirty" if the repository has uncommited change
-    # /!\ HACK for searx/searx: ignore searx/brand.py and utils/brand.env
+    # /!\ HACK for searx/searx: ignore utils/brand.env
     git update-index -q --refresh
-    if [ ! -z "$(git diff-index --name-only HEAD -- | grep -v 'searx/brand.py' | grep -v 'utils/brand.env')" ]; then
```

Comment:
The change makes sense but I'm confused by the existing code:
* this line is using the git command line.
* line 149, 155, 160 uses `GIT_URL`

Can we have a scenario where `GIT_URL` is different from `git remote get-url origin` ?

Reply:
I guess you missunderstand the lines of code; this is "grep -v" to suppress changes in the brand-files when calling a git diff.

Did I answered your question?

Reply:
Nope :-)
* Line 128 uses `git` so it assumes that the current directory is a git repository most probably clone from somewhere: `git remote get-url origin` gives the origin.
* Line 159 uses `git rev-parse HEAD` to get the git commit
* Line 160 uses `GIT_URL` to get the git URL.

According to line 159 and 160, either `GIT_URL` equals `git remote get-url origin`, either we can have an incoherence.

So my question is: when `GIT_URL` is different from `git remote get-url origin`. 

Anyway this change was made before this PR.